packages/adaptive_scaffold/pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: adaptive_scaffold
+description: Widgets to easily build adaptive layouts, including navigation elements.
+version: 0.0.1
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Aadaptive_scaffold
```

Comment:
I missed this before: is this actually the label this package will be using? Usually our package labels are of the form "p: <name of package>". Is there a reason to deviate from that here?